.gitignore

```diff
@@ -120,3 +120,5 @@ app.*.symbols
 !**/ios/**/default.perspectivev3
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
 !/dev/ci/**/Gemfile.lock
+/packages/flutter/example/linux/flutter/ephemeral/
+/packages/flutter/example/macos/Flutter/ephemeral/
```

Comment:
The gitignores are currently are all over the place and need to be consolidated. Could you just add an entry to the root gitignore under the existing "# macOS" heading:

```
**/macos/Flutter/ephemeral
```

Reply:
Done.

Remove line.